### PR TITLE
Bump zkvm and bonsai-sdk version to 1.1.0-rc.2

### PR DESCRIPTION
**Pull request by SchmErik**
This change updates zkvm  and `bonsai-sdk` to versions to `1.1.0-rc.2`.

### CODE REVIEW

**Comment by SchmErik** [comment]
Converting to draft until lights go green, after that, I'll publish crates and merge this PR after I do so successfully

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,26 @@
...
+
+### 🛠 Fixes
+* Released Rust toolchain `r0.1.79.0-1`. This toolchain fixes a crash in the guest that appears when enabling the heap allocator in combination with `thread_local!` in the zkVM guest.
+* Clean up `risc0_core::scope!` macro used to measure performance. (#2217)
+* Clippy now works by default on host-side code for rust projects that contain risc0 guest code. To run Clippy on the host-side code for risc0 projects, specify `RISC0_SKIP_BUILD=1` as a part of the Clippy command. (#2210)
+* Add linter to website (#2096)
+
+### ⚡️ Features
+* Add `cargo risczero datasheet` command. This enables users to run performance benchmarks on their machines locally. When run on arm64 macOS, the datasheet is generated using GPU proving. When run on Linux, this command will the prover on CPU by default. (#2196)
+* Move `risc0-circuit-bigint` to risc0 repo. This is a foundational component for our accelerators. More documentation will follow as we solidify this crate and accelerators. (#2201)
```
Comment (flaub) [inline CHANGELOG.md:17]:
```suggestion
* Add the `risc0-circuit-bigint` crate. This is a foundational component for our accelerators. More documentation will follow as we solidify this crate and accelerators. (#2201)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,26 @@
...
+* Speed up building guest programs by 20x. The `risc0-build` crate is responsible for building guest programs and is invoked from Rust build scripts which is run using a dev profile by default. This results in significant overhead when computing the image ID of the guest. To speed up the build process call out to the `r0vm` binary in order to compute the image ID. (#2208)
+
+### 🛠 Fixes
+* Released Rust toolchain `r0.1.79.0-1`. This toolchain fixes a crash in the guest that appears when enabling the heap allocator in combination with `thread_local!` in the zkVM guest.
+* Clean up `risc0_core::scope!` macro used to measure performance. (#2217)
+* Clippy now works by default on host-side code for rust projects that contain risc0 guest code. To run Clippy on the host-side code for risc0 projects, specify `RISC0_SKIP_BUILD=1` as a part of the Clippy command. (#2210)
+* Add linter to website (#2096)
+
+### ⚡️ Features
+* Add `cargo risczero datasheet` command. This enables users to run performance benchmarks on their machines locally. When run on arm64 macOS, the datasheet is generated using GPU proving. When run on Linux, this command will the prover on CPU by default. (#2196)
```
Comment (flaub) [inline CHANGELOG.md:16]:
```suggestion
* Add `cargo risczero datasheet` command. This enables users to run performance benchmarks on their machines locally. (#2196)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,26 @@
 # CHANGELOG
 
 ## Next (upcoming release)
+## [v1.1.0-rc.2 (2024-08-22)](https://github.com/risc0/risc0/releases/tag/v1.1.0-rc.2)
+### 🔥 Performance Improvements
+* Improve GPU performance by 28%. For more information checkout our [reports page](https://reports.risczero.com/benchmarks#Linux-nvidia_rtx_3090_ti-succinct). (#2211)
+* Speed up building guest programs by 20x. The `risc0-build` crate is responsible for building guest programs and is invoked from Rust build scripts which is run using a dev profile by default. This results in significant overhead when computing the image ID of the guest. To speed up the build process call out to the `r0vm` binary in order to compute the image ID. (#2208)
```
Comment (flaub) [inline CHANGELOG.md:7]:
```suggestion
* Speed up null builds of guest programs by 20x. The `risc0-build` crate is responsible for building guest programs and is invoked from Rust build scripts which is run using a dev profile by default. This results in significant overhead when computing the image ID of the guest. To speed up the build process call out to the `r0vm` binary in order to compute the image ID. (#2208)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,26 @@
...
+
+### 🛠 Fixes
+* Released Rust toolchain `r0.1.79.0-1`. This toolchain fixes a crash in the guest that appears when enabling the heap allocator in combination with `thread_local!` in the zkVM guest.
+* Clean up `risc0_core::scope!` macro used to measure performance. (#2217)
```
Comment (flaub) [inline CHANGELOG.md:11]: This is a new addition, not a fix

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,6 +1,26 @@
...
+* Add linter to website (#2096)
+
+### ⚡️ Features
+* Add `cargo risczero datasheet` command. This enables users to run performance benchmarks on their machines locally. When run on arm64 macOS, the datasheet is generated using GPU proving. When run on Linux, this command will the prover on CPU by default. (#2196)
+* Move `risc0-circuit-bigint` to risc0 repo. This is a foundational component for our accelerators. More documentation will follow as we solidify this crate and accelerators. (#2201)
+* Add examples and improve verbiage for datasheet. (#2242)
+* Add blog link and use a cleaner mermaid theme for code examples on developer documentation website. (#2190)
+### 🚨 Breaking Changes
+* The poseidon hashing function has been removed from the prover. This hashing function has been superseded by poseidon2 in the prover for a while now. For those who are using default prover configurations and the public API, this has absolutely no impact. For those who are using the old poseidon hash function to generate risc0 proofs, it is highly encouraged to switch to poseidon2 for improved performance.
```
Comment (flaub) [inline CHANGELOG.md:21]:
```suggestion
* Add blog link and use a cleaner mermaid theme for code examples on developer documentation website. (#2190)

### 🚨 Breaking Changes

* Drop the obsolete and unused poseidon hashing function, which has been superseded by poseidon2.
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,6 +1,26 @@
 # CHANGELOG
 
 ## Next (upcoming release)
+## [v1.1.0-rc.2 (2024-08-22)](https://github.com/risc0/risc0/releases/tag/v1.1.0-rc.2)
+### 🔥 Performance Improvements
+* Improve GPU performance by 28%. For more information checkout our [reports page](https://reports.risczero.com/benchmarks#Linux-nvidia_rtx_3090_ti-succinct). (#2211)
+* Speed up building guest programs by 20x. The `risc0-build` crate is responsible for building guest programs and is invoked from Rust build scripts which is run using a dev profile by default. This results in significant overhead when computing the image ID of the guest. To speed up the build process call out to the `r0vm` binary in order to compute the image ID. (#2208)
+
+### 🛠 Fixes
+* Released Rust toolchain `r0.1.79.0-1`. This toolchain fixes a crash in the guest that appears when enabling the heap allocator in combination with `thread_local!` in the zkVM guest.
```
Comment (flaub) [inline CHANGELOG.md:10]:
```suggestion
### 🛠 Fixes

* Released Rust toolchain `r0.1.79.0-1`. This toolchain fixes a crash in the guest that appears when enabling the heap allocator in combination with `thread_local!` in the zkVM guest.
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,6 +1,26 @@
 # CHANGELOG
 
 ## Next (upcoming release)
+## [v1.1.0-rc.2 (2024-08-22)](https://github.com/risc0/risc0/releases/tag/v1.1.0-rc.2)
+### 🔥 Performance Improvements
+* Improve GPU performance by 28%. For more information checkout our [reports page](https://reports.risczero.com/benchmarks#Linux-nvidia_rtx_3090_ti-succinct). (#2211)
```
Comment (flaub) [inline CHANGELOG.md:6]:
```suggestion
## [v1.1.0-rc.2 (2024-08-22)](https://github.com/risc0/risc0/releases/tag/v1.1.0-rc.2)

### 🔥 Performance Improvements

* Improve GPU performance by 28%. For more information checkout our [reports page](https://reports.risczero.com/benchmarks#Linux-nvidia_rtx_3090_ti-succinct). (#2211)
```

**Review by flaub — CHANGES_REQUESTED** [review]
Want to get more info about the bonsai-sdk version bump

**Review by flaub — APPROVED** [review]

**Pull request merged**